maps/sccv_horizon/code/sccv_horizon_unittest.dm

```diff
@@ -12,7 +12,7 @@
 		/area/supply/dock,
 		/area/turbolift,
 		/area/mine,
-		/area/horizon/exterior,
+		/area/horizon/,
```

Comment:
making every horizon-relative area excempt from environment unit tests is probably not a good idea.